styles/rules.css

```diff
@@ -0,0 +1,26 @@
+.rule-area{
+    width: 50rem;
+    height: 30rem;
+    background-color: var(--background-gray);
+    border-radius: var(--radius-sm);
+    padding: var(--space-md);
+}
+
+.txt-important{
+    color: var(--pastel-yellow);
+}
+
+.rules-cta{
+    gap: var(--space-md);
+}
+
+.rules-cta>button{
+    width: 15rem;
+    border: none;
+    border-radius: 0%;
+    background-color: var(--pastel-red);
```

Comment:
You can just remove border-radius.

Reply:
Hi, the border radius is defined here is to override the default value that comes from the component library I have used. So it's serving a purpose here.